code-challenges.js

```diff
@@ -30,35 +44,65 @@ var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "tem
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const onlyNumbers = (array) => {
+    let newArr = []
+    newArr = array.sort((a,b) => a - b).filter(value => typeof value === "number")
+     return newArr 
 
-
+}
```

Comment:
good job! you can even do this all in one line if you'd like ;)
ex: `return array.sort((a,b) => a - b).filter(value => typeof value === "number")`